### ISSUE TITLE
Enhance kubernetes_ingress with ingress_class_name attribute

### ISSUE THREAD

**Issue by lkishalmi** [hashicorp/terraform-provider-kubernetes#1218]
### Description
Please add an ingress_class_name to the kubernetes_ingress resource, as it is specified at Kubernetes 1.18.

### Potential Terraform Configuration

```hcl
resource "kubernetes_ingress" "example" {
  ...
  spec {
    ingress_class_name = "external-lb"
    tls {
        ...
    }
    ...
  }
}
```

### References
https://kubernetes.io/blog/2020/04/02/improvements-to-the-ingress-api-in-kubernetes-1.18/#putting-it-all-together

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Comment by MathyV** [comment]
I think this was already resolved in #1056 , not sure why it isn't showing up in the help though.

**Comment by DrFaust92** [comment]
Adds missing docs here https://github.com/hashicorp/terraform-provider-kubernetes/pull/1236
